index.html

```diff
@@ -39,6 +40,221 @@ <h3>LET'S CONNECT</h3>
                     </ul>
                 </section>
         </div>
+         <!-- work -->
+         <div id="work_1">
+            <div class="project">
+                <img src="images/header/tonic.png" alt="tonic">
+            </div>
+            <h4 class="project_detail">Tonic</h4>
+            <div class="job_detail">
+                <h4 class="canopy">CANOPY</h4>
+                <p class="dot">•</p>
+                <p class="job">Back End Dev</p>
+                <p class="dot">•</p>
+                <p class="yrs">2015</p>
+            </div>
+            <div class="read">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="progm_skills">
+                <ul>
+                    <li class="skill_list1">
+                        <button>html</button>
+                    </li>
+                    <li class="skill_list2">
+                        <button>CSS</button>
+                    </li>
+                    <li class="skill_list3">
+                        <button>javaScript</button>
```

Comment:
[OPTIONAL] As stated in the project requirement below, you are supposed to create two sections at these stage of the project, one section for the `works/projects` and another section for the `about me`. 
 
![Screenshot from 2022-09-24 12-11-24](https://user-images.githubusercontent.com/65233860/192097230-5503370a-0c23-4a88-8e25-599798091143.png)

Reply:
### I used ul li tag for html,css and js for this section
and earlier reviewer told me to use h tag in work_section bcz earlier I was using div.
i changed everything which she said 
except hover bcz that is not so compulsory.
few hours left for deadline,plz don't block for that reason.